.github/workflows/label_stale.yml

```
@@ -0,0 +1,24 @@
+name: Label and close stale PRs and Issues
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs at midnight UTC every day
+
+jobs:
+  stale-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Seek and destroy stale PRs and Issues
+        uses: actions/stale@v4
+        with:
+          stale-pr-message: 'This pull request seems to be stale as there have been no changes in 14 days, please make changes within 7 days or the PR will be closed. If you believe this is a mistake, please inform a development team member on Discord.'
+          close-pr-message: 'This pull request was marked as stale, yet no changes have been observed in the specified time. Please feel free to open a new pull request when your project has been updated, however this pull request has been closed.'
```

Feels like it reads a bit better
```suggestion
          close-pr-message: 'This pull request has not received any updates since being marked stale, and as such is now being automatically closed. Please feel free to re-open this pull request or open a new one once you have new updates.'
```